conn/spi/spi.go

```diff
@@ -103,6 +103,20 @@ type Packet struct {
 // io.Reader.
 type Conn interface {
 	conn.Conn
+	// TxPackets does multiple operations over the SPI bus.
+	//
+	// The maximum number of bytes can be limited depending on the driver. Query
+	// conn.Limits.MaxTxSize() can be used to determine the limit.
+	//
+	// If the last packet has KeepCS:true, the behavior is undefined. The CS line
+	// will likely not stay asserted. This is a driver limitation.
+	TxPackets(p []Packet) error
+}
+
+// ConnConfig is the interface to be provided to device drivers.
+//
+// The driver calls DevParams() to retrieve a configured connection.
```

Comment:
By "driver", I meant "device driver", but really, it is the "consumer" of the connection, the slave end. It'd be good to find a word that carries the right semantic and could then be used thoroughly in the code base.